File: codegen/testing/test/testdata/simple-range-pp/nodejs/simple-range.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const bucket: aws.s3.Bucket[] = [];
for (const range = {value: 0}; range.value < 10; range.value++) {
    bucket.push(new aws.s3.Bucket(`bucket-${range.value}`, {website: {
        indexDocument: `index-${range.value}.html`,
    }}));
}

File: codegen/testing/test/testdata/simple-range-pp/dotnet/simple-range.cs
using System.Collections.Generic;
using Pulumi;
using Aws = Pulumi.Aws;

return await Deployment.RunAsync(() => 
{
    var bucket = new List<Aws.S3.Bucket>();
    for (var rangeIndex = 0; rangeIndex < 10; rangeIndex++)
    {
        var range = new { Value = rangeIndex };
        bucket.Add(new Aws.S3.Bucket($"bucket-{range.Value}", new()
        {
            Website = new Aws.S3.Inputs.BucketWebsiteArgs
            {
                IndexDocument = $"index-{range.Value}.html",
            },
        }));
    }
});

